cmd/snap/cmd_snap_op.go

```diff
@@ -1107,6 +1112,8 @@ func init() {
 			// TRANSLATORS: This should not start with a lowercase letter.
 			"cohort": i18n.G("Install the snap in the given cohort"),
 			// TRANSLATORS: This should not start with a lowercase letter.
+			"ignore-validation": i18n.G("Ignore validation by other snaps blocking the refresh"),
```

Comment:
s/refresh/installation/ ?

Reply:
Ah yes, I copied it from refresh help but it needs changing here, thanks.